lib/crypto.js

```diff
@@ -60,7 +60,8 @@ const {
 const {
   createSecretKey,
   createPublicKey,
-  createPrivateKey
+  createPrivateKey,
+  KeyObject,
```

Comment:
Nit: Trailing comma.

Reply:
Trailing commas were requested by @sam-github

Reply:
@sam-github I remember various discussions around this, however not a single one that resulted in a decision to move towards trailing commas. Did that happen recently?

Reply:
@tniessen https://github.com/nodejs/node/pull/26438#discussion_r262193693

Reply:
AFAICT, we've been moving towards trailing commas, for the decrease of irrelevant diff noise. I can't track all the discussions in nodejs, though, maybe I'm not abreast of current style. Its certainly not the kind of stuff that should be discussed per PR :-(, I'd hope it was documented somewhere, but I can't find it.

Reply:
We've had multiple discussions around trailing commas and never come to a decisive conclusion, but my impression is that there is a general preference **for** trailing commas, but our source code seems to predominantly **not** use trailing commas.

I've been slowly implementing trailing commas for multi-line arrays only but it's....slow.... and only arrays, not objects (just because arrays will be less churn than objects so I'm doing them first, but even just arrays would be a lot of churn.....)

Reply:
FWIW I don't like trailing commas.

Reply:
And I still agree with @lpinca.

Reply:
Yeah, I avoided describing it as "consensus" for that reason. There are definitely folks who disagree. And it's why we have the _status quo_ of a mish-mash of trailing commas and no trailing commas.

I'd be fine with the mish-mash--just accept that we can do either--but unfortunately, people leave nits asking for trailing commas (such as @bnoordhuis) and other people ask for them to be removed (@tniessen above). So I'd much prefer to codify one or the other in our tooling so that the nits stop. I don't care which one we go with, but previous conversations make it seem like the "yes to trailing commas" folks are either more numerous or more vocal, so....

Reply:
This is not the right place to discuss about this but I think we should either always use trailing commas or never. Mixing them is the worst thing we can do in my opinion.

Reply:
> This is not the right place to discuss about this but I think we should either always use trailing commas or never.

I'm fine changing it / leaving it, i just follow the collaborator's guideline since 3 weeks now (#26200) to get something this silly through. Getting ready to give up and actually live with a [hack](https://github.com/panva/jose/blob/v0.10.0/lib/help/key_object.js) instead.

Reply:
Yes, sorry this is silly.

Reply:
they are already a mix, lets leave alone for now, with or without, doesn't matter for this PR.